src/evo/evonotificationinterface.cpp

```diff
@@ -10,12 +10,12 @@
 void EvoNotificationInterface::InitializeCurrentBlockTip()
 {
     LOCK(cs_main);
-    UpdatedBlockTip(chainActive.Tip(), nullptr, IsInitialBlockDownload());
+    deterministicMNManager->UpdatedBlockTip(chainActive.Tip());
 }
 
 void EvoNotificationInterface::UpdatedBlockTip(const CBlockIndex *pindexNew, const CBlockIndex *pindexFork, bool fInitialDownload)
 {
-    deterministicMNManager->UpdatedBlockTip(pindexNew);
+    // !TODO background thread updates
```

Comment:
can just remove this function if it's not used anymore.

Reply:
it will be used by `llmq::quorumManager` and `llmq::quorumDKGSessionManager`